src/pages/components/Content/Content.jsx

```diff
@@ -0,0 +1,106 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useState, useMemo } from 'react';
+import { DragDropContext, Draggable } from 'react-beautiful-dnd';
+import StrictModeDroppable from '../../../kit/components/StrictModeDroppable/StrictModeDroppable';
+import { reorderTasks } from '../../../slices/tasksSlice';
+
+import Title from '../../../kit/components/Title/Title';
+import Container from '../../../kit/components/Container/Container';
+import TaskNew from '../TaskNew/TaskNew';
+import ListControls from '../ListControls/ListControls';
+import styles from './Content.module.scss';
+import TaskCreated from '../TaskCreated/TaskCreated';
+
+const Content = () => {
+  const tasks = useSelector((state) => state.tasksList.tasks.filter((t) => !t.deleted));
+  const tasksOrder = useSelector((state) => state.tasksList.tasksOrder);
+
+  const dispatch = useDispatch();
+  const reorderTasksHandler = ((newTaskIds) => {
+    dispatch(reorderTasks(newTaskIds));
+  });
+
+  const [buttonsActivity, setButtonsActivity] = useState({
+    all: true,
```

Comment:
Looks a bit complicated. I'd suggest storing the selected filter as a string.

Reply:
changed - filter is stored as string